src/js/index.js

```diff
@@ -24,6 +24,22 @@ import ApplicationUtil from './utils/ApplicationUtil';
 
 let domElement = document.getElementById('application');
 
+// TODO: Implement loader that can concat many sprites into a single one
```

Comment:
Not sure I understand this. Is this sprite sheet for CNVS? We can just use a regular loader and import the sprites where we need them couldn't we?